apps/demos/src/app/rx-angular-pocs/cdk/utils/rxjs/operators/select.ts

```diff
@@ -156,7 +168,11 @@ export function select<T>(
     if (!opOrMapFn || opOrMapFn.length === 0) {
       return state$.pipe(stateful());
     } else if (isStringArrayGuard(opOrMapFn)) {
-      return state$.pipe(stateful(pluck(...opOrMapFn)));
+      return state$.pipe(
+        stateful(
+          map((state) => opOrMapFn.reduce((acc, key) => acc?.[key], state))
```

Comment:
Question, do we want to use our `safePluck` implementation here instead?

Reply:
it's in the pocs, let's not care about it